src/Renci.SshNet/Abstractions/ThreadAbstraction.cs

```diff
@@ -10,24 +10,17 @@ internal static class ThreadAbstraction
         /// <param name="millisecondsTimeout">The number of milliseconds for which the thread is suspended.</param>
         public static void Sleep(int millisecondsTimeout)
         {
-#if FEATURE_THREAD_SLEEP
             System.Threading.Thread.Sleep(millisecondsTimeout);
-#elif FEATURE_THREAD_TAP
-            System.Threading.Tasks.Task.Delay(millisecondsTimeout).GetAwaiter().GetResult();
-#else
-            #error Suspend of the current thread is not implemented.
-#endif
         }
 
         public static void ExecuteThreadLongRunning(Action action)
         {
-#if FEATURE_THREAD_TAP
```

Comment:
I propose to leave this as is. Eventually, we'll need to change this as we need the thread or task that we started for our message listener in order to avoid thread leaks (there's an issue for this).